core/LocOffsets.h

```diff
@@ -27,6 +27,14 @@ struct LocOffsets {
         ENFORCE_NO_TIMER(exists());
         return beginLoc == endLoc;
     }
+
+    LocOffsets() : beginLoc(INVALID_POS_LOC), endLoc(INVALID_POS_LOC) {}
```

Comment:
If you initialize these with curly braces, the compiler will raise an error if any numeric coercion occurs in the future.